plugins/sphinx/sphinx_status

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# -*- sh -*-
+: <<=cut
+
+=head1 NAME
+
+sphinx - Munin plugin to display Sphinx status
+
+=head1 CONFIGURATION
+
+Config file must be readable by your munin user.
+  [sphinx_*]
+  user sphinx
+
+optional is category, default is empty
+  [sphinx_*]
+  env.category mysql
+
+=head1 AUTHOR
+
+martin fuxa <root@unihost.cz>
+
+=head1 LICENSE
+
+CC
```

Comment:
This (CC) looks like a reference to "Creative Commons", where there are multiple licenses, none if which are abbreviated "CC" . Could you specify or clarify the license?